client/components/standard-deposit-button/modal.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button, Modal } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { formatCurrency, formatExplicitCurrency } from 'utils/currency';
+import './style.scss';
+
+interface StandardDepositModalProps {
+	availableBalance: AccountOverview.Balance;
+	depositDelayDays: number;
+	onClose: () => void;
+	onSubmit: () => void;
+	inProgress: boolean;
+}
+
+const StandardDepositModal = ( {
+	availableBalance,
+	depositDelayDays,
+	onClose,
+	onSubmit,
+	inProgress,
+}: StandardDepositModalProps ): JSX.Element => {
+	/* translators: %s: amount representing the available monetary balance, %s: number of business days */
+	const description = sprintf(
+		__(
+			'Ready to get paid? Your available balance of %s will be deposited to your bank account within %s business days.',
+			'woocommerce-payments'
+		),
+		formatExplicitCurrency(
```

Comment:
I've used the explicit currency format here (e.g. `$14.15 AUD`), which diverges from the design but is more informative for the merchant IMO.